app/services/editor-commands/commands/center-items.ts

```diff
@@ -0,0 +1,32 @@
+import { ModifyTransformCommand } from './modify-transform';
+import { Selection } from 'services/selection';
+
+export enum ECenteringType {
+  Screen = 'screen',
+  Horizontal = 'horizontal',
+  Vertical = 'vertical',
+}
+
+export class CenterItemsCommand extends ModifyTransformCommand {
```

Comment:
Why we need to define a special subset of commands like `CenterItemsCommand` if we already have one general Transform command. Looks like this adds more boilerplate code

Reply:
It's definitely more boilerplate.  One reason is that this command has a different description that is shown in the UI for undo/redo operations.  I tried to encapsulate as much shared logic as possible in the `ModifyTransformCommand` base command.  This also allowed me to leverage all of our existing transform helper functions that operate on selections/scene-items directly, instead of having to generate a new transform and pass it the `SetTransform` command.